### PR TITLE
Implement fluent for some Menu files

### PR DESCRIPTION
**Pull request by Visne**
Implement fluent for

BlockGameMenu
SpaceVillainArcadeMenu
CommunicationsConsoleMenu
ConstructionMenuPresenter
ConfigurationMenu
WiresMenu
LatheQueueMenu

### CODE REVIEW

File: Content.Client/Construction/ConstructionMenuPresenter.cs [hunk 1]
```diff
@@ -181,7 +181,7 @@ private void PopulateCategories()
             var uniqueCategories = new HashSet<string>();
 
             // hard-coded to show all recipes
-            uniqueCategories.Add(Loc.GetString("All"));
+            uniqueCategories.Add(Loc.GetString("construction-presenter-all"));
```
Comment (RemieRichards) [inline Content.Client/Construction/ConstructionMenuPresenter.cs:184]: `construction-presenter-category-all`?

File: Content.Client/GameObjects/Components/Configuration/ConfigurationMenu.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ public ConfigurationMenu(ConfigurationBoundUserInterface owner)
 
             _inputs = new List<(string name, LineEdit input)>();
 
-            Title = Loc.GetString("Device Configuration");
+            Title = Loc.GetString("configuration-device");
```
Comment (RemieRichards) [inline Content.Client/GameObjects/Components/Configuration/ConfigurationMenu.cs:29]: I like the `-title` suffix you had for some of the other menus, it'd be nice if that was used consistently!

File: Content.Client/GameObjects/Components/Wires/WiresMenu.cs [hunk 3]
```diff
@@ -147,14 +147,14 @@ public WiresMenu(WiresBoundUserInterface owner)
                 {
                     (_nameLabel = new Label
                     {
-                        Text = Loc.GetString("Wires"),
+                        Text = Loc.GetString("wires-wires"),
```
Comment (RemieRichards) [inline Content.Client/GameObjects/Components/Wires/WiresMenu.cs:150]: You know what's wrong with this

File: Content.Client/GameObjects/Components/Wires/WiresMenu.cs [hunk 4]
```diff
@@ -147,14 +147,14 @@ public WiresMenu(WiresBoundUserInterface owner)
...
                         FontOverride = font,
                         FontColorOverride = StyleNano.NanoGold,
                         VerticalAlignment = VAlignment.Center,
                     }),
                     (_serialLabel = new Label
                     {
-                        Text = Loc.GetString("DEAD-BEEF"),
+                        Text = Loc.GetString("wires-dead-beef"),
```
Comment (RemieRichards) [inline Content.Client/GameObjects/Components/Wires/WiresMenu.cs:157]: this is a tricky one, `DEAD-BEEF` is hexadecimal, it's not *really* in english at all.
I probably wouldn't translate this at all.

**Comment by RemieRichards** [comment]
In general good, but left some points to go over

File: Content.Client/Arcade/BlockGameMenu.cs [hunk 5]
```diff
@@ -160,7 +160,7 @@ public BlockGameMenu(BlockGameBoundUserInterface owner)
 
             _newGameButton = new Button
             {
-                Text = Loc.GetString("New Game"),
+                Text = Loc.GetString("blockgame-new-game"),
```
Comment (RemieRichards) [inline Content.Client/Arcade/BlockGameMenu.cs:163]: Sorry, didn't mention this yesterday but, maybe make these `blockgame-button-new-game`, `blocgkame-button-scoreboard` etc.

Just in general try to include a *little* bit more context

File: Content.Client/Arcade/BlockGameMenu.cs [hunk 6]
```diff
@@ -230,7 +230,7 @@ public BlockGameMenu(BlockGameBoundUserInterface owner)
                 VerticalAlignment = VAlignment.Center
             };
 
-            gameOverMenuContainer.AddChild(new Label{Text = Loc.GetString("Gameover!"),Align = Label.AlignMode.Center});
+            gameOverMenuContainer.AddChild(new Label{Text = Loc.GetString("blockgame-game-over"),Align = Label.AlignMode.Center});
```
Comment (RemieRichards) [inline Content.Client/Arcade/BlockGameMenu.cs:233]: `blockgame-msg-game-over` maybe?

File: Content.Client/Arcade/BlockGameMenu.cs [hunk 7]
```diff
@@ -288,7 +288,7 @@ public BlockGameMenu(BlockGameBoundUserInterface owner)
                 VerticalAlignment = VAlignment.Center
             };
 
-            menuContainer.AddChild(new Label{Text = Loc.GetString("Highscores")});
+            menuContainer.AddChild(new Label{Text = Loc.GetString("blockgame-highscores")});
```
Comment (RemieRichards) [inline Content.Client/Arcade/BlockGameMenu.cs:291]: labels get `-label-` in their message Ids?

File: Content.Client/Command/CommunicationsConsoleMenu.cs [hunk 8]
```diff
@@ -24,12 +24,12 @@ public CommunicationsConsoleMenu(CommunicationsConsoleBoundUserInterface owner)
...
             IoCManager.InjectDependencies(this);
 
-            Title = Loc.GetString("Communications Console");
+            Title = Loc.GetString("communications-console-title");
             Owner = owner;
 
             _messageInput = new LineEdit
             {
-                PlaceHolder = Loc.GetString("Announcement"),
+                PlaceHolder = Loc.GetString("communications-console-announcement"),
```
Comment (RemieRichards) [inline Content.Client/Command/CommunicationsConsoleMenu.cs:32]: `communications-console-announcement-placeholder`?

**Review by RemieRichards — CHANGES_REQUESTED** [review]
Sorry I didn't really consider any of these yesterday.

Also thinking maybe these all need `menu` in them somewhere? mainly for the case where an item has a menu, it could get confusing whether a message refers to the item (i.e. an interaction) or the item's menu.
@PJB3005 ^ ?

**Comment by Visne** [comment]
Oh god it's been three weeks
Ready for review again

**Review by PaulRitter — APPROVED** [review]

**Comment by PaulRitter** [comment]
gtm with @RemieRichards approval

**Comment by RemieRichards** [comment]
PJ's clearly not listening so, implement the last thing I said above and I'll sign off on it.

**Comment by Visne** [comment]
> 
> 
> PJ's clearly not listening so, implement the last thing I said above and I'll sign off on it.

@RemieRichards Done.

**Review by RemieRichards — APPROVED** [review]

**Pull request merged**